src/module/canvas/token/object.ts

```diff
@@ -123,6 +123,62 @@ class TokenPF2e extends Token<TokenDocumentPF2e> {
         return flankingBuddies.some((b) => onOppositeSides(this, b, flankee));
     }
 
+    /** Overrides _drawBar() to also draw pf2e variants of normal resource bars (such as temp health) */
+    protected override _drawBar(number: number, bar: PIXI.Graphics, data: TokenResourceData): void {
+        if (!canvas.dimensions) return;
+
+        const actor = this.document.actor;
+        const isHealth = data.attribute === "attributes.hp" && actor?.attributes.hp;
+
+        if (!isHealth) {
+            super._drawBar(number, bar, data);
+            return;
```

Comment:
```suggestion
            return super._drawBar(number, bar, data);
```

Reply:
Surprised you're ok with return void. I know core does it, but it always felt like a bad idea. That might be my previous programming exp talking.